core/src/main/scala/kafka/server/ControllerServer.scala

```diff
@@ -42,6 +42,7 @@ import org.apache.kafka.raft.RaftConfig
 import org.apache.kafka.raft.RaftConfig.AddressSpec
 import org.apache.kafka.server.authorizer.Authorizer
 import org.apache.kafka.server.common.ApiMessageAndVersion
+import org.apache.zookeeper.server.quorum.QuorumPeerConfig.ConfigException
```

Comment:
Hmm, it looks like this is the wrong ConfigException. The one you should be importing is `org.apache.kafka.common.config.ConfigException`

Reply:
Thanks for catching that. It did feel like this was at an odd place.